apps/server/project.json

```diff
@@ -68,6 +68,13 @@
       "options": {
         "command": "npx prisma generate --schema apps/server/prisma/schema.prisma"
       }
+    },
```

Comment:
Since the GraphQL schema is generated by the `graphql` library, it's best to leverage Nx for that rather than duplicating the target.

You can mark the `graphql` library as a dependency of `server`, and then add `^graphql-codegen` to the `dependsOn` array of `build` here.